network_provisioning/examples/thread_prov/main/idf_component.yml

```diff
@@ -0,0 +1,7 @@
+dependencies:
```

Comment:
I think we should add the example version at this place as well.
As done here https://github.com/espressif/idf-extra-components/blob/9e3bf69c3f992d66dccb0f6dd76b90062af76c55/coap/examples/coap_client/main/idf_component.yml#L1

Same for the wifi_prov example

Reply:
Done